nodes/usb_cam_node.cpp

```diff
@@ -328,7 +328,8 @@ class UsbCamNode
     enable_auto_reset_exposure_ = true;
     auto_reset_exposure_timer_ = node_.createTimer(
       ros::Duration(auto_reset_exposure_period_),
-      boost::bind(&UsbCamNode::checkAutoResetExposure, this, _1)
+      boost::bind(&UsbCamNode::checkAutoResetExposure, this, _1),
+      auto_reset_exposure_period_ <= 0
```

Comment:
Oh actually, are you ok with this timer triggering immediately? Before you had a sleep, but now `auto_reset_exposure_period_` is 0 so this timer will go off almost immediately.

Reply:
yeah you're right, added different period constant for one shot case